src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stringsArr = sourceString.split(';');
+  const res = {};
+
+  stringsArr.forEach((str) => {
+    const clSrt = str.trim();
+
+    const ruleParam = clSrt.split(':');
```

Comment:
The variable `ruleParam` could be renamed to `keyValuePair` for better clarity, as it represents a key-value pair split from the string.